robolectric/src/test/java/org/robolectric/shadows/ShadowAccessibilityServiceTest.java

```diff
@@ -0,0 +1,53 @@
+package org.robolectric.shadows;
+
+import android.accessibilityservice.AccessibilityService;
+import android.view.accessibility.AccessibilityEvent;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.TestRunners;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.robolectric.Shadows.shadowOf;
+
+@RunWith(TestRunners.WithDefaults.class)
+public class ShadowAccessibilityServiceTest {
+  private MyService service ;
+  private ShadowAccessibilityService shadow;
+
+  @Before
+  public void setup() {
+    service = Robolectric.setupService(MyService.class);
+    shadow = shadowOf(service);
```

Comment:
Rather than using a subclass of `AccessibilityService`, I think it would be more realistic to use `RuntimeEnvironment.application.getSystemService(Context.ACCESSIBILITY_SERVICE)`, and then get the shadow of that.